src/openforms/submissions/tasks/__init__.py

```diff
@@ -41,6 +42,10 @@ def on_post_submission_event(submission_id: int, event: PostSubmissionEvents) ->
     # just set a submission reference (if it hasn't already been set)
     pre_registration_task = pre_registration.si(submission_id, event)
 
+    pre_registration_component_group = pre_registration_component_group_task.si(
+        submission_id
+    )
```

Comment:
Could you change the naming to keep it aligned with the other tasks?
```suggestion
    pre_registration_component_group_task = pre_registration_component_group.si(
        submission_id
    )
```